wofi-wifi-menu.sh

```diff
@@ -21,7 +21,7 @@ fi
 
 LIST=$(nmcli --fields "$FIELDS" device wifi list | sed '/^--/d')
 # For some reason wofi always approximates character width 2 short... hmmm
-RWIDTH=$(($(echo "$LIST" | head -n 1 | awk '{print length($0); }')+2))
+RWIDTH=$(($(echo "$LIST" | head -n 1 | awk '{print length($0); }')*10))
```

Comment:
@charlie-haley I am testing the changes locally, sadly still without a wifi adapter, but I am getting some errors when trying to run the script right now. I think it might be due to the line 24 changes

![image](https://user-images.githubusercontent.com/49439044/80727176-a9e9f180-8b05-11ea-8792-00ec51445871.png)

Are you having a different experience? What was the motivation for this change?

Reply:
So, the reason for doing that was, $RWIDTH was returning 44 for me. The `width` parm on rofi is a percentage (if the value is under 100) and by default the `width` param on wofi is in pixles. Obviously 44 pixels is tiny. and 44% is far too large

Reply:
Interestingly, I'm getting a similar error, but slightly different.. I'll take a look into why this is happening.

I also realise a better solution will be needed for $RWIDTH, 440px is quite large for people who aren't using high resolution screens.

![image](https://user-images.githubusercontent.com/27928201/80731970-86be4280-8b03-11ea-845e-be55375bd32f.png)

Reply:
![image](https://user-images.githubusercontent.com/27928201/80733546-c423cf80-8b05-11ea-87ff-71489f8e5e5e.png)
Despite the warnings, it is working and looking good!

Reply:
The error you are getting might come from either the config file of wofi or an upstream issue. I am getting a similar one as well

![image](https://user-images.githubusercontent.com/49439044/80733278-c558fa80-8b0d-11ea-932d-749f8992ae55.png) (as can also be seen in the first pic of mine)

The critical errors seem to actually be doing something, as my wofi looks like this now: 
![image](https://user-images.githubusercontent.com/49439044/80733451-03561e80-8b0e-11ea-83c4-b4866ea233fd.png)

> I also realise a better solution will be needed for $RWIDTH, 440px is quite large for people who aren't using high resolution screens.

I agree with this, the difference between the two programs is big in this regard.

What do you think about merging the changes regarding the syntax in this PR, closing the first issue and creating two more issues related to the above? Would you be interested joining as a contributor? :)

Reply:
Go for it! I'll create 2 new issues for the $RWIDTH issue and the gtk warnings